.gradle.log

```diff
@@ -0,0 +1,5 @@
+[INFO] Scanning for projects...
```

Comment:
Please add this file to .gitignore and remove from PR

Reply:
Will do